roles/common/tasks/kernel-tuning.yml

```diff
@@ -22,10 +22,14 @@
   notify:
     - apply-sysctl
 
+- stat: path=/etc/default/grub
+  register: grub_defaults
+
 - name: "Kernel boot parameters: disable quiet boot"
   lineinfile: dest=/etc/default/grub
               regexp="^GRUB_CMDLINE_LINUX_DEFAULT="
               line="GRUB_CMDLINE_LINUX_DEFAULT=\"\""
+  when: grub_defaults.stat.exists == True | bool
```

Comment:
it should be:

`when: grub_defaults.stat.exists | bool`

But actually I think I was pre-mature. Because you're using the stat module, what you'll be getting back IS a boolean, so it's fine.  It was just the check against `serial_console.enabled` that needed the bool.

For here you can just do:

`when: grub_defaults.stat.exists`